portfolio/src/main/webapp/index.html

```diff
@@ -48,13 +48,19 @@ <h2>Academic Achievements</h2>
     </ul>
 
     <p> View my transcript <a href="images/AcademicTranscript.pdf">here</a></p>
-    <button onclick="getFromServlet()">Click me!</button>
-    <div id="saying-container"></div>
     <br>
+    <h3>Give feedback!</h3>
     <form action="data" method="POST">
         <input type="text" name="user-comment">
         <input type="submit"/>
     </form>
+    <br>
+    <form>
+    <p> Number of comments to show</p>
+    <input type = "number" id="load-comments" onchange="getFromServlet()" value ="0" max="10" min="0"> 
```

Comment:
FYI, the "this" keyword actually contains the event within the onchange string.  So something like "getFromServlet(this.value)" would pass the value to the method without the method having to know which element it came from.